app/views/schools/register_mentor_wizard/email_address.html.erb

```diff
@@ -1,12 +1,18 @@
-<% page_data(title: "What is #{@mentor.full_name}'s email address?", error: @wizard.current_step.errors.present?, backlink_href: schools_register_mentor_wizard_review_mentor_details_path) %>
+<% 
+  page_data(
+    title: "What is #{@mentor.full_name}'s email address?", 
+    error: @wizard.current_step.errors.present?, 
+    backlink_href: @mentor.email.present? ? schools_register_mentor_wizard_check_answers_path : schools_register_mentor_wizard_review_mentor_details_path
```

Comment:
Not sure if this works for all the cases. 
What if the user completes steps but didn't get to the check answers step and then comes to this step via `Back` buttons? Would we be redirecting them to check-answers prematurely?

Reply:
I think back links generally are a bigger topic that we should fix at the wizard (lib) level rather than address individually.

It's super complex and we're at risk of just recreating the browser back button which is pointless.